### PR TITLE
Windows Installer tweaks

### PR DESCRIPTION
**Pull request by chfast**
The installer can create start menu shortcuts for GUI applications **but** that information is not connected with optional components installation. So the shortcut will be created for application that was not installed. I made all the GUI applications required.

### CODE REVIEW

File: cmake/EthExecutableHelper.cmake [hunk 1]
```diff
@@ -186,15 +186,26 @@ macro(eth_nsis)
 		set(CPACK_COMPONENT_RLP_GROUP "CLI")
 		set(CPACK_COMPONENT_ABI_GROUP "CLI")
 
+		set(CPACK_COMPONENT_ALETHZERO_REQUIRED TRUE)
```
Comment (arkpar) [inline cmake/EthExecutableHelper.cmake:208]: Could we make it optional but enabled by default instead?

**Comment by arkpar** [comment]
Could you also add ethkey to the installer?

File: cmake/EthExecutableHelper.cmake (hunk 1 above)
Comment (chfast) [inline cmake/EthExecutableHelper.cmake:208]: No if you don't want broken links.

**Comment by chfast** [comment]
Sure.

**Comment by chfast** [comment]
ethkey added. PATH modification disabled. Default installation folder is "Ethereum".

File: cmake/EthExecutableHelper.cmake [hunk 2]
```diff
@@ -168,13 +168,30 @@ macro(eth_nsis)
...
+		set(CPACK_PACKAGE_FILE_NAME "Ethereum")
 		set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "The Ethereum Toolset")
 		set(CPACK_PACKAGE_VENDOR "ethereum.org")
 		set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
 		set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
 		set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION})
-		set(CPACK_GENERATOR "NSIS")
+		set(CPACK_NSIS_DISPLAY_NAME "Ethereum")
+		set(CPACK_PACKAGE_INSTALL_DIRECTORY "Ethereum")
+		set(CPACK_NSIS_HELP_LINK "https://github.com/ethereum/cpp-ethereum")
```
Comment (arkpar) [inline cmake/EthExecutableHelper.cmake]: This repo is about to be deleted

Comment (chfast) [inline cmake/EthExecutableHelper.cmake]: Good spot.

**Comment by chfast** [comment]
Help link updated.

File: cmake/EthExecutableHelper.cmake [hunk 3]
```diff
@@ -168,13 +168,30 @@ macro(eth_nsis)
...
 		set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "The Ethereum Toolset")
 		set(CPACK_PACKAGE_VENDOR "ethereum.org")
 		set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
 		set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
 		set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION})
-		set(CPACK_GENERATOR "NSIS")
+		set(CPACK_NSIS_DISPLAY_NAME "Ethereum")
+		set(CPACK_PACKAGE_INSTALL_DIRECTORY "Ethereum")
+		set(CPACK_NSIS_HELP_LINK "https://www.gitbook.com/book/gavofyork/turboethereum")
+		set(CPACK_NSIS_URL_INFO_ABOUT "https://github.com/ethereum/cpp-ethereum")
```
Comment (arkpar) [inline cmake/EthExecutableHelper.cmake]: This also needs updating

**Comment by chfast** [comment]
About link updated.

**Pull request merged**